src/apps/Permissions/EntitySelector.js

```diff
@@ -75,6 +96,14 @@ class EntitySelector extends React.Component {
       items.splice(-1, 0, 'Any account')
     }
 
+    if (includeBurnEntity) {
+      items.splice(-1, 0, 'Burn role')
+    }
+
+    if (includeUnassignedEntity) {
+      items.splice(-1, 0, 'Unassign role')
```

Comment:
I don't think we need this—we already have the "remove manager" and "revoke permission" actions exposed.